packages/nvlsm/nvlsm.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Start NVLink Subnet Manager
+RequiresMountsFor=/etc
```

Comment:
```suggestion
```

Reply:
I rebuilt and tested without this change and the behavior looks the same.